package.json

```diff
@@ -21,6 +21,8 @@
     "@react-stately/collections": "3.4.1",
     "@react-stately/tabs": "3.1.1",
     "@react-types/shared": "3.13.1",
+    "@types/chroma-js": "2.1.4",
+    "chroma-js": "2.4.2",
```

Comment:
Is it used or just for future use?

Reply:
The `chroma.alpha()` function is used in`theme.tsx` for a few colors. They were previously created using a transform that only honorable understood.

Reply:
Makes sense :+1: